### PR TITLE
Feature/message limits

### PR DESCRIPTION
**Pull request by AkhileshNegi**
- Added API for message counts exchanged in the last 24 hrs
- Added test cases for API
- Updated documentation

### CODE REVIEW

File: lib/glific/stats.ex [hunk 1]
```diff
@@ -449,4 +449,25 @@ defmodule Glific.Stats do
...
+  def get_bsp_message_count(organization_id) do
+    start_date = Timex.shift(DateTime.utc_now(), days: -1)
+    end_date = DateTime.utc_now()
+
+    %{messages: messages} =
+      Stat
+      |> where([s], s.organization_id == ^organization_id)
+      |> where([s], s.period == "hour")
+      |> where([s], s.inserted_at > ^start_date and s.inserted_at <= ^end_date)
+      |> select([s], %{
```
Comment (pankaj-ag) [inline lib/glific/stats.ex]: I don't think get_bsp_message_count name makes sense here.

get_message_count(org_id, period, opts) 

you can send start date and end date in the opts itself.

**Review by pankaj-ag — CHANGES_REQUESTED** [review]

File: lib/glific/stats.ex [hunk 2]
```diff
@@ -449,4 +449,26 @@ defmodule Glific.Stats do
...
+  @spec get_message_count(non_neg_integer(), Keyword.t()) :: {:ok, non_neg_integer()}
+  def get_message_count(organization_id, opts \\ []) do
+    default_start_date = Timex.shift(DateTime.utc_now(), days: -1)
+    start_date = Keyword.get(opts, :start_date, default_start_date)
+    end_date = Keyword.get(opts, :end_date, DateTime.utc_now())
+
+    %{messages: messages} =
+      Stat
+      |> where([s], s.organization_id == ^organization_id)
+      |> where([s], s.period == "hour")
```
Comment (pankaj-ag) [inline lib/glific/stats.ex]: Can we send the period also as an argument? Just check my last comment.

**Review by pankaj-ag — CHANGES_REQUESTED** [review]

File: lib/glific/stats.ex (hunk 1 above)
Comment (AkhileshNegi) [reply lib/glific/stats.ex]: done

File: lib/glific/stats.ex (hunk 2 above)
Comment (AkhileshNegi) [reply lib/glific/stats.ex]: I have made requested changes

**Pull request closed without merging**